monitoring/configuring-the-monitoring-stack.adoc

```diff
@@ -140,6 +140,13 @@ include::modules/monitoring-disabling-grafana.adoc[leveloffset=+1]
 
 * See xref:../monitoring/configuring-the-monitoring-stack.adoc#preparing-to-configure-the-monitoring-stack[Preparing to configure the monitoring stack] for steps to create monitoring config maps
 
+// Disabling the local Alertmanager
+include::modules/monitoring-disabling-the-local-alertmanager.adoc[leveloffset=+1]
+
+.Additional resources
+* See the link:https://prometheus.io/docs/alerting/latest/alertmanager/[Prometheus Alertmanager documentation] for more information about Alertmanager.
+* See xref:../monitoring/managing-alerts.html[Managing alerts] for more information about managing and configuring alerts in the {product-title} monitoring stack.
```

Comment:
Consider using just the link. Using helper text around the link is no longer required/encouraged.

Reply:
```suggestion
* See xref:../monitoring/managing-alerts.adoc#managing-alerts_managing-alerts[Managing alerts] for more information about managing and configuring alerts in the {product-title} monitoring stack.
```
It looks like the build is complaining about this xref. I believe we need to reformulate this xref to use the following structure instead. `../monitoring/<filename>.adoc#<bookmark>_<context attribute>

Reply:
Fixed. Thanks.